wls-infomanagement-service/src/test/java/de/muenchen/oss/wahllokalsystem/infomanagementservice/configuration/SecurityConfigurationTest.java

```diff
@@ -64,4 +85,91 @@ void accessUnsecuredResourceSwaggerUiThenOk() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @Nested
+    class Konfiguration {
```

Comment:
die einzelnen Endpunkte auch in weitere Subclasses verlagern? Oder lässt sich daraus vielleicht sogar ein parametrisierter Test bauen oder für andere Controller zur Verfügung stellen?

Reply:
Also vermutlich kann man einiges vereinfachen:

<details><summary>Ein CodeSnipper eines Versuches</summary>

```
        static Stream<Arguments> getAuthenticatedTestArguments() {
            return Stream.of();
        }

        static Stream<Arguments> getAnonymousTestArguments() {
            return Stream.of(createArgumentsForUnauthorized("/businessActions/konfiguration/ABSCHLUSSTEXT"),
                    createPostArguments("/businessActions/konfiguration/ABSCHLUSSTEXT", status().isUnauthorized()),
                    createArgumentsForUnauthorized("/businessActions/konfiguration"),
                    createArgumentsForUnauthorized("/businessActions/kennbuchstaben"),
                    createGetArguments("/businessActions/konfigurationUnauthorized/WILLKOMMENSTEXT", status().isNoContent()));
        }

        private static Arguments createArgumentsForUnauthorized(final String path) {
            return createGetArguments(path, status().isUnauthorized());
        }

        private static Arguments createGetArguments(final String path, final ResultMatcher resultMatcher) {
            return createArguments(path, get(path), resultMatcher);
        }

        private static Arguments createPostArguments(final String path, final ResultMatcher resultMatcher) {
            return createArguments(path, post(path).with(csrf()).contentType(MediaType.APPLICATION_JSON), resultMatcher);
        }

        private static Arguments createArguments(final String name, final MockHttpServletRequestBuilder requestBuilder, final ResultMatcher resultMatcher) {
            return Arguments.of(name, requestBuilder, resultMatcher);
        }
```

Allerdings wird es um einiges komplizierter wenn man noch Stubbing definieren muss oder einen Button im Request benötigt. Daher würde ich daraus ein gesondertes Issue machen.

</details>

Reply:
ausgelagert in neues Issue #232 

eine Unterteilung in weitere Subklassen halte ich auch nicht unbedingt für erforderlich. Mal sehen was ein Reviewer meint 😄